File: Game Randomizer.cpp
#include <iostream>
#include <vector> 
#include <string>
#include <ctime>
#include <algorithm>

using namespace std;





int main() {

	
	bool cont = true;
	srand(time(NULL));

	do {

		system("cls");

		vector<string> games = {"Overwatch", "Cyberpunk 2077", "GTA V", "Forza Horizon 3", "Forza Horizon 5", "Control", "The Legend Of Zelda: TOTK", "Himno", "Star Citizen"};
	
		games.push_back("Sea of Thieves");
		games.push_back("Just Cause 3");
		games.push_back("Assetto Corsa Competizione");
		games.push_back("Kerbal Space Program");


		/* 	

		games.push_back("Papers, Please");
		games.push_back("Jedi: Fallen Order");
		games.push_back("Rocket League");
		games.push_back("Wolfenstein"); 
		
		*/

		vector <int> printed;

		for (int i = 0; i <= games.size(); i++) {
			
			int j = rand() % games.size();


			if (!count(printed.begin(), printed.end(), j)) {

				printed.push_back(j);
				cout << "\n\t" << i+1 << ". " << games[j];

			}

			else {

				j = rand() % games.size();
				
				if (printed.size() == games.size()) {
				
					break;
				}
				
				else {

					i--;
				} 
			}
		}

		char respuesta;

		do {

			cout << endl << "\n¿Quieres volver a tirar? (S/N)";
			cin >> respuesta;
			respuesta = toupper(respuesta);
		} while (respuesta != 'S' && respuesta != 'N');

		if (respuesta == 'N') {cont = false;}

	} while (cont);

	return 0;

 } 

File: Game Randomizer.py

import random




while True:
    games = ["Overwatch", "Cyberpunk 2077", "GTA V", "Forza Horizon 3", "Forza Horizon 5", "Control", "The Legend Of Zelda: TOTK", "HearthStone"]

    games.append("Sea of Thieves")
    games.append("Outer Wilds")
    games.append("Mirror's Edge Catalyst")
    games.append("The Finals")
    games.append("Just Cause 3")
    games.append("Assetto Corsa Competizione")
    games.append("Kerbal Space Program")
    games.append("Forza Horizon 4")
    games.append("Half-Life")
    games.append("SteamWorld Dig 2")
    games.append("Star Citizen")
    games.append("BioShock")
    games.append("MultiVersus")
    games.append("Super Mario Odessey")

    
    #games.append("Himno")
    #games.append("Papers, Please")
    #games.append("Jedi: Fallen Order")
    #games.append("Rocket League")
    #games.append("Wolfenstein")

    nGames = len(games)

    for i in range(nGames):
        j = random.randint(0, len(games) -1)
        print("\t %d. %s" % (i + 1, games[j]))
        games.remove(games[j])
    
    

    while True:
        
        resp = input(str("\n\nDo you want to roll again? (Y/N): "))
        resp = resp.upper()

        if (resp == 'Y') ^ (resp == 'N'):
            break
    
    if resp == 'N':
        break
    

    